### PR TITLE
Add the bones

### PR DESCRIPTION
**Pull request by kylefiedler**
- Adds Bourbon / Neat / Bitters
- Adds basic Jekyll stuff
- Adds html for main layout and start of homepage

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta content="width=device-width, initial-scale=1.0" name="viewport">
+    <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon" />
+    <link rel="stylesheet" href="/stylesheets/style.css" type="text/css" me     dia="all" />
```
Comment (mborsare) [inline _layouts/default.html]: extra spaces in `me      dia`

File: _layouts/default.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+          <li class="dropdown">
+            <a href="#">Philly Tech Info</a>
+            <ul>
+              <li><a href="">Resources</a></li>
+              <li><a href="">Events</a></li>
+              <li><a href="">Coworking</a></li>
+              <li><a href="">Startups</a></li>
+              <li><a href="">Investors</a></li>
+              <li><a href="">Jobs</a></li>
+              <li><a href="">Incubators &amp; Accelorators</a></li>
```
Comment (mborsare) [inline _layouts/default.html]: sp. Accelerators

File: _layouts/default.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+      <footer>
+        <article>
+          <h1>Site design &amp; development by</h1>
+          <h2>thoughtbot</h2>
+          <p>Our clients hire us to design and build their products. We focus on user outcomes and simplicity. We write code for iOS, Android, and the web. <a href="#">Learn more about thoughtbot</a></p>
+        </article>
+
+        <section>
+          <h1><span>PSL</span>Philly Startup Leaders</h1>
+          <p>This site and it’s code are open source and licenced under MIT license. View the source.</p>
```
Comment (mborsare) [inline _layouts/default.html]: I'm guessing `View the source.` should be a link

File: _layouts/default.html [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+            <li>
+              <a href="#">Philly Tech Info</a>
+              <ul>
+                <li><a href="">Resources</a></li>
+                <li><a href="">Events</a></li>
+                <li><a href="">Coworking</a></li>
+                <li><a href="">Startups</a></li>
+                <li><a href="">Investors</a></li>
+                <li><a href="">Jobs</a></li>
+                <li><a href="">Incubators &amp; Accelorators</a></li>
```
Comment (mborsare) [inline _layouts/default.html]: sp. Accelerators

**Comment by mborsare** [comment]
Minor stuff – otherwise looking good.

File: Gemfile [hunk 5]
```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'github-pages'
+gem 'sass'
```
Comment (whmii) [inline Gemfile]: if your using the right dir structure, yaml heads, etc… Jekyll will handle all your sass for you. You could drop this and just let Jekyll's asset handling take care of it.
http://jekyllrb.com/docs/assets/

File: _layouts/default.html [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+    <meta charset="UTF-8">
+    <meta content="width=device-width, initial-scale=1.0" name="viewport">
+    <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon" />
+    <link rel="stylesheet" href="/stylesheets/style.css" type="text/css" me     dia="all" />
+    <title>{{ page.title }}</title>
+  </head>
+
+  <body>
+    <header>
+      <h1><span>PSL</span>Philly Startup Leaders</h1>
```
Comment (whmii) [inline _layouts/default.html]: Abbreviation tag? I've never had cause to use one. Now might be your chance!
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr

**Comment by whmii** [comment]
just some fun additions, otherwise :sake:

File: Gemfile (hunk 5 above)
Comment (kylefiedler) [inline Gemfile]: @whmii I'm using the Jekyll asset management so it's probably covered in the 'github-pages' gem.

**Pull request merged**